tools/packaging/packaging.psm1

```diff
@@ -106,7 +106,7 @@ function Start-PSPackage {
             -not $PSModuleRestoreCorrect -or                        ## Last build didn't specify '-PSModuleRestore' correctly
             $Script:Options.Runtime -ne $Runtime -or                ## Last build wasn't for the required RID
             $Script:Options.Configuration -ne $Configuration -or    ## Last build was with configuration other than 'Release'
-            $Script:Options.Framework -ne "netcoreapp2.0")          ## Last build wasn't for CoreCLR
+            $Script:Options.Framework -ne "netcoreapp2.1")          ## Last build wasn't for CoreCLR
```

Comment:
I see other places using `netcoreapp2.0`. They all need to be updated.
```powershell
PS:103> dir *.psm1 -Recurse | sls -SimpleMatch netcoreapp2.0

test\tools\OpenCover\OpenCover.psm1:627:        [parameter()]$PowerShellExeDirectory =
"${script:psRepoPath}/src/powershell-win-core/bin/CodeCoverage/netcoreapp2.0/win7-x64/publish",
tools\packaging\packaging.psm1:2221:        New-MSIPackage -Verbose -ProductCode (New-Guid) -ProductSourcePath
'.\src\powershell-win-core\bin\Debug\netcoreapp2.0\win7-x64\publish' -ProductTargetArchitecture x64 -ProductVersion
'1.2.3'
build.psm1:560:    # publish netcoreapp2.0 reference assemblies
```

Reply:
Fixed.